frontend/src/model/inferencing/EditableInference.tsx

```diff
@@ -65,12 +66,17 @@ export default function EditableInference({ inference }: EditableInferenceProps)
   }
 
   const handleSubmit = async () => {
+    if (!isPortNumber(port)) {
+      return setErrorMessage('Please use a valid port number.')
```

Comment:
Again, would be useful to explain what a valid port is. `'Port number must be in the range 1-65535'`